docs/conf.py

```diff
@@ -5,13 +5,7 @@
 #
 # Configuration file for the Sphinx documentation builder.
 
-import os
-import sys
-import shutil
```

Comment:
These changes are conflicts with #100. If you want to merge this before #100, please tell me 👍 
https://github.com/onnx/sklearn-onnx/pull/100/files#diff-85987f48f1258d9ee486e3191495582dL10

Reply:
That's ok. I'll probably give up on this one as it does not seem to reach a consensus.